ha/core/health_monitor/health_monitord.py

```diff
@@ -74,6 +75,8 @@ def __init__(self, singleton_check: bool = False):
         self._confstore = ConfigManager.get_confstore()
         self._rule_manager = MonitorRulesManager()
         self._event_consumer = self._get_consumer()
+        # Initialize conf store
+        ConftStoreSearch()
```

Comment:
Why is this needed in Health monitor ?

Reply:
Not required, but added for all services can avail this.
Removed.